### ISSUE TITLE
Collapse all sets automatically when dragging

### ISSUE THREAD

**Issue by stuartcusackie** [statamic/ideas#883]
It's quite tricky to drag and order large field sets into in a replicator field. 

I had the same problem on a custom app I built using Vue draggable / sortable. Simple solution is to automatically collapse all sets on dragstart, and then automatically re-expand them on dragend. It makes for a much nicer dragging experience.

Thanks.

**Comment by AndreasSchantl** [comment]
I wanted to open app an issue for exactly the same, but would have a different approach. 

If it is a replicator, a set within a blueprint, etc. – display up and down arrows instead, to move a set one by one.

**Comment by AndreasSchantl** [comment]
I wanted to open up an issue for the same thing a few days ago!

I would have a different approach for it though. Instead of collapsing, why not display an up and down arrow instead to move sets within replicators, blueprints and so on?

**Comment by stuartcusackie** [comment]
Arrows are okay but they don't help much when you have 20 items in a grid.

Here's a rough GIF of what I mean:
![ezgif com-gif-maker (6)](https://user-images.githubusercontent.com/20210934/195101120-e2da9c49-5b59-4ee9-b80b-b2f98f5c55db.gif)

**Comment by AndreasSchantl** [comment]
Looks good. If it works that well as in your GIF, then I'd be more than happy. Hearing that all the time from customers and seeing myself struggle sometimes if big sets are to be dragged around.

**Comment by robdekort** [comment]
Agreed. Would be great if you could collapse grid stacked sets and maybe even have auto collapsing like you suggest. Either this one or #140 can probably be closed.

**Comment by robdekort** [comment]
Now we have Statamic V4 I figured the replicator takes a lot less space with all the `+` buttons being hidden now. So I converted my stacked grid to a replicator with one set. It's A LOT easier to use now. The only difference from a stacked grid vs the v4 replicator is that you can't change the `add` label. It's the just the `+` button now. Which is fine, because with the replicator in accordion mode you can see the field label most of the times so you know what you're adding. _Thought_: the stacked grid might might be redundant.
